src/components/Product/Product.js

```diff
@@ -76,10 +79,21 @@ function Product() {
   };
 
   const format = (name) => {
-    return name.toLowerCase().replace(/[\W]+/g, '');
+    return name.toLowerCase().replace(/^\s+|\s+$/g, '');
```

Comment:
Cool that they organized this. 💛

Reply:
For future occasions: 

Consider giving Regular Expressions a meaningful name, for example:

`const LOWERCASE_NAMES_ONLY = /..../`;